apps/dashboard/src/@3rdweb-sdk/react/hooks/useDashboardContractMetadata.tsx

```diff
@@ -27,6 +27,18 @@ export function useDashboardContractMetadata(contract: ThirdwebContract) {
           getCompilerMetadata(contract).catch(() => undefined),
         ]);
 
+      /**
+       * Handle the case where the contract.thirdweb.com endpoint returns a very long name
+       * example url: https://contract.thirdweb.com/metadata/324/0x66E298335c5992d599132582De8eaAF75348e6e6
+       * In that case, we split the solidity name on ":" and take the second part
```

Comment:
fwiw this looks like a potential contract api bug, would track it - for now makes sense